index.html

```diff
@@ -113,11 +113,12 @@ <h3>Terminology</h3>
 <p><dfn>Bidi isolation</dfn> often needs to be applied to a range of text in order to prevent the automatic rules of the Unicode Bidirectional Algorithm incorrectly ordering that content in relation to the surrounding text. For example, numbers following right-to-left text in memory are automatically positioned to the left of that text by the Bidi Algorithm, but sometimes need to appear to the right. Another example occurs when lists of RTL items occur in a LTR sentence: the Bidi Algorithm will automatically assume that the order of items in the list should be "3 ,2 ,1", but actually what's needed is "1, 2, 3". In HTML, bidi isolation can be applied to a range of text by enclosing it in an element with a <code class="kw" translate="no">dir</code> attribute. In plain text there are Unicode formatting characters that can do the job. These mechanisms remove unwanted 'spillover effects'.</p>
-<p>Unicode code points are associated with properties relating to text direction. Generally, Arabic and Hebrew letters have a strong RTL direction, whereas Latin and Han characters have a strong LTR direction. Other characters, such as punctuation, only have a weak intrinsic directionality, and the actual directionality is determined according to the context in which they are found. <dfn>First-strong detection</dfn> is an algorithm that looks for the first strongly-directional character in a string, and then uses that to guess at the appropriate base direction for the string as a whole.</p>
+<p> <dfn>First-strong detection</dfn> is an algorithm that looks for the first strongly-directional character in a string, and then uses that to guess at the appropriate base direction for the string as a whole. Unicode code points are associated with properties relating to text direction: generally, Arabic and Hebrew letters have a strong RTL direction, whereas Latin and Han characters have a strong LTR direction. Other characters, such as punctuation, only have a weak intrinsic directionality, and the actual directionality is determined according to the context in which they are found.</p>
+<p><dfn>Base direction</dfn>  determines whether items of content will be arranged <em>left-to-right</em> or <em>right-to-left</em>, relative to each other in bidirectional text. The focus of the Unicode Bidirectional Algorithm (UBA) is the way individual adjacent characters <em>of the same direction</em> are arranged relative to each other. However, when there are clumps of both LTR and RTL character sequences, or when there are weak characters such as punctuation, the relative placement of these items depends on the surrounding directional context (the base direction). </p>
```

Comment:
This paragraph doesn't make sense to me, particularly the emphasized part about same direction? Perhaps:

> *Base direction* determines the starting point and general progression of content, either *left-to-right* or *right-to-left*, relative to each other in bidirectional text. The focus of UBA is the way in which adjacent logical characters are arranged relative to each other visually. When characters are of the same direction, this is primarily driven by the characters themselves. However, when there are clumps of both LTR and RTL character sequences, or when there are weak characters such as punctuation, the relative placement of these items depends on the surrounding directional context (which stems from the base direction).

Reply:
Here's my suggestion (i'll add to the update i'm making):

> Base direction determines the general arrangement and progression of content when bidirectional text is displayed. The UBA is primarily focused on arranging adjacent characters, based on character properties. Base direction works at a higher level, and dictates (a) the visual order and direction in which runs of strongly-typed LTR and RTL character are displayed, and (b) where there are weakly-typed characters such as punctuation, the placement of those items relative to the other content.